resources/windows/msi/service.mcollective.wxs

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <Component Id='MCOService' Guid="7601FCEA-90B3-CC69-6A69-4087FBC7292D" Win64="$(var.Win64)">
+      <File Id="RubyWExe" KeyPath="yes" Source="stagedir\sys\ruby\bin\rubyw.exe" />
```

Comment:
@jpogran This will be the "true" filename - with however, we express the top of the source or work directory tree - the file is copied as-is to the work/build area.